tests/comfy_cli/test_tracking.py

```
@@ -0,0 +1,105 @@
+    """End-to-end: init_tracking() writes the string "False"/"True", and track_event honors it.
+
+    Regression for a prior bug where track_event used config_manager.get(), which returned
+    the raw string "False" (a truthy value), so disabling via this code path had no effect.
+    """
+
+    def test_disable_is_respected_by_track_event(self, tracking_module):
+        tracking_module.init_tracking(False)
+        tracking_module.track_event("some_event")
+        tracking_module.mp.track.assert_not_called()
+
+    def test_enable_is_respected_by_track_event(self, tracking_module):
```

Pylint (4.0.5)

[convention] Missing function or method docstring

(C0116)